modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/frontend/data/set/view/table/ObjectRelationshipsTableFDSView.java

```diff
@@ -48,6 +48,10 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"objectDefinitionName2", "related-object"
 		).add(
 			"type", "type"
+		).add(
+			"reverse", "hierarchy",
+			fdsTableSchemaField -> fdsTableSchemaField.setContentRenderer(
+				"hierarchyDataRenderer")
```

Comment:
I believe that Paulo did this way because the return is going to be the permanent way (with this new column) and he put the condition with !FF and the old way before the return statement so is going to be easier to delete in the future...